### PR TITLE
Albedo support

### PR DESCRIPTION
**Pull request by ralphilius**
Add functionality to sign with Albedo in txsigner as per #545

https://github.com/tyvdh/stellar-quest-bounties/issues/11

### CODE REVIEW

File: src/actions/transactionSigner.js [hunk 1]
```diff
@@ -160,5 +164,28 @@ export function signWithFreighter(txXDR, txNetwork) {
...
+          signedTx: result.signed_envelope_xdr
+        });
+      })
+      .catch(e => {
+        dispatch({
+          type: ALBEDO_WALLET_SIGN_ERROR,
+          error: e.message,
+        });
+      })
+
```
Comment (quietbits) [inline src/actions/transactionSigner.js:187]: Nitpick: extra line break here.

File: src/components/TransactionSigner.js [hunk 2]
```diff
@@ -216,6 +235,13 @@ class TransactionSigner extends React.Component {
                     {freighterwalletMessage}
                   </OptionsTablePair>
                 )}
+                <OptionsTablePair label="Albedo">
+                    <button
+                      className="s-button"
+                      onClick={() => { dispatch(signWithAlbedo(xdr, networkPassphrase === Networks.TESTNET ? "TESTNET" : "PUBLIC")) }}
+                    >Sign with Albedo</button>
+                    {albedowalletMessage}
+                  </OptionsTablePair>
```
Comment (quietbits) [inline src/components/TransactionSigner.js:244]: Nitpick: indentation seems a bit off.

**Review by quietbits — COMMENTED** [review]
Nice PR, thank you very much! 🙌 Just left a couple of very tiny comments.

**Comment by ralphilius** [comment]
> Nice PR, thank you very much! 🙌 Just left a couple of very tiny comments.

I've made those changes 😃

**Comment by quietbits** [comment]
Thank you, @ralphilius ! 🙂 We'll publish this very soon.

**Review by piyalbasu — APPROVED** [review]

**Comment by shanzzam** [comment]
@tyvdh will you review this please

**Comment by tyvdh** [comment]
It's got the 👍  from me!

**Pull request merged**